pkg/rewards/claim.go

```diff
@@ -154,13 +237,59 @@ func Claim(cCtx *cli.Context, p utils.Prompter) error {
 	logger.Info("Validating claim proof...")
```

Comment:
this should now have earner in the log message so that you can display for which earner you are validating